### ISSUE TITLE
FCC database location changed?  

### ISSUE THREAD

**Issue by jtashiro** [tgorgdotcom/locast2plex#199]
Was experiencing timeout during locast2plex startup, and found that the URL for FCC database was moved.

I updated my local copy of lib/stations.py to correct the address.

-    facility_url = 'https://transition.fcc.gov/ftp/Bureaus/MB/Databases/cdbs/facility.zip'
+    facility_url = 'https://transition.fcc.gov/Bureaus/MB/Databases/cdbs/facility.zip'

Any objections to externalizing this and put into the config.ini file?

**Comment by tgorgdotcom** [comment]
Hmmm, opening this on the browser seems to be working, as well as the old http URL.  I'm wondering if we're hammering the FCC server and we are being actively blocked or something.  If that's the case, it may be good for me to change the frequency we hit the server, as well as adding some additional things to help.

**Comment by jtashiro** [comment]
Agreed the browser approach works.  Running curl -v shows a http 301 permanent move to the URL without 'ftp', so I took that as the proper path.  Seems OK now with that URL.

**Comment by tgorgdotcom** [comment]
That is a good point though.  If it's a 301, then maybe it is a real change.  I'll make the switch in code and see if we run into any additional issues.

**Comment by tgorgdotcom** [comment]
Just updated the code and released v0.6.5
